help center: Update emoji reactions documentation. (#23343)
zulip/zulip: templates/zerver/help/emoji-reactions.md

1. Select an emoji. Type to search, use the arrow keys, or click on an emoji
   with your mouse.

!!! tip ""
    To add multiple reactions without closing the emoji picker, hold the
    <kbd>Shift</kbd> key while selecting emoji.

{tab|mobile}

1. Long press on a message.

1. Select **Add a reaction** in the menu that appears.

1. Select an emoji. Type to search, or tap the emoji you'd like to use.

{end_tabs}

## Add or remove an existing reaction

{start_tabs}

{tab|desktop-web}

> 1. Click on an existing reaction to add or remove your vote.

Reviewer: Not sure if the following would be an improvement, but how about:

```suggestion
1. Click on an existing emoji response to add or remove your reaction.
```

Author: Mmm, I think I'd rather not introduce "response" as additional terminology, but adding "emoji" for clarity seems good.

Author: So:

> 1. Click on an existing emoji reaction to add or remove your reaction.

(and the parallel change below)

!!! tip ""
    To make it easy to see which reactions you have added, they are
    highlighted in a different color.

{tab|mobile}

> 1. Tap on an existing reaction to add or remove your reaction.

Reviewer: and same here:

```suggestion
1. Tap on an existing emoji response to add or remove your reaction.
```

!!! tip ""

>     To make it easy to see which reactions you have added, they are
>     highlighted in a different color.

Reviewer: Maybe we should say this more concisely. How about something like:

```suggestion
    Reactions you have added are highlighted in a different color.
```
